Libraries/Opc.Ua.Server/Session/SessionManager.cs

```diff
@@ -631,6 +631,16 @@ private void MonitorSessions(object data)
         }
         #endregion
 
+        #region Protected Methods reading some private attributes
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+        protected IServerInternal Server { get { return m_server; } }
+        protected int MaxRequestAge => m_maxRequestAge;
+        protected int MaxBrowseContinuationPoints => m_maxBrowseContinuationPoints;
+        protected int MaxHistoryContinuationPoints => m_maxHistoryContinuationPoints;
```

Comment:
These all come from configuration and are all readonly properties. I see no need to expose them, they could be read just from ApplicationConfiguration

Reply:
the need arises from the fact that I want to derive from SessionManager. This fails if you do not make these attributes protected, because in the virtual Session CreateSession these private attributes are used in the call to the constructor of Session. I think this call to the constructor should be changed, but I'm confused by that coding...